bot.py

```diff
@@ -12,6 +14,30 @@
 # target_channel_id = 722609125950750771 # testing
 activity = True
 
+@bot.event
+async def on_message(message):
+    if message.author == client.user:
+        return
+    match = re.search((?:\b|\s+)(?:ope)(?:\s+|\b), message.content.lower)
```

Comment:
This would also miss if there were multiple opes in the message I think, I googled a bit and https://stackoverflow.com/a/17268979/9044659 seems useful to what we're doing.